tmva/tmva/inc/TMVA/RTensor.hxx

```diff
@@ -276,8 +277,8 @@ public:
       Iterator operator++(int) { auto tmp = *this; operator++(); return tmp; }
       Iterator& operator--() { fGlobalIndex--; return *this; }
       Iterator operator--(int) { auto tmp = *this; operator--(); return tmp; }
-      Iterator operator+(difference_type rhs) const { return Iterator(fTensor, fGlobalIndex + rhs); }
```

Comment:
These operator + and - should be defined as const functions since they return a new type.

Reply:
made the change, thanks for pointing that out.